test/fixtures/pull_requests.yml

```diff
@@ -0,0 +1,15 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
Let's add and configure rspec gem for testing. Please prepare a new branch after this'd be merged.

Reply:
Have I done right?

Reply:
Yep, but this should be done after merging your previous changes.
Also @sergiiss has already done this in his PR.
So, please, revert your changes.